charts/camunda-platform-alpha-8.8/values.yaml

```diff
@@ -2426,28 +2426,26 @@ core:
     ## @param core.index.prefix if specified, defines web apps index prefix in Elasticsearch/OpenSearch. Note, for Zeebe index prefix, use "global.elasticsearch.prefix".
     prefix: ""
 
-  archiver:
-    ## @param core.archiver.enabled if false, there is no archiver which will continually create historical indices.
-    enabled: true
```

Comment:
FYI: This was removed on purpose as we expect this property to always be configured.